src/lib/components/spinner/index.tsx

```diff
@@ -6,6 +6,7 @@ export interface SpinnerProps {
 
 const Spinner = ({ size = 's' }: SpinnerProps) => (
   <div
+    aria-live='polite'
```

Comment:
Do I understand correctly, that aria-live needs [some text or message that changes dynamically and should be announced](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-live#description)? If so, what would it be in this case?